src/config.ts

```diff
@@ -13,6 +13,7 @@ export function resolveOptions(options: Partial<VitePWAOptions>, viteConfig: Res
   const {
     srcDir = 'public',
     outDir = viteConfig.build.outDir || 'dist',
+    inline = true,
```

Comment:
Maybe `inlineScript` could be more clear, what do you think?

Reply:
you are right - fixed this